e2e/rbd.go

```diff
@@ -675,6 +676,36 @@ var _ = Describe("RBD", func() {
 					e2elog.Failf("expected volSnapContentName %q got %q", content.Name, volSnapContentName)
 				}
 
+				// make sure we had unset the PVC metadata on the rbd image created
+				// for the snapshot
+				pvcName, stdErr, err := execCommandInToolBoxPod(f,
+					fmt.Sprintf("rbd image-meta get %s --image=%s %s",
+						rbdOptions(defaultRBDPool), imageList[0], pvcNameKey),
+					rookNamespace)
+				if err == nil || !strings.Contains(err.Error(), "command terminated with exit code 2") ||
```

Comment:
this is a very long if-check, maybe use a helper function as it is repeated often?